src/assignments/5-loop.js

```diff
@@ -18,9 +25,17 @@ number = 125
 output example
 sum=8
 
+
 */
 // start here writing the code for program2
 
+const num = prompt("enter number")
```

Comment:
check whether the num is the number or letter. if the number, do the calculation; otherwise, give the alert. 
https://stackoverflow.com/questions/15047140/javascript-prompt-number-and-continue-prompting-if-answer-is-wrong 
Example  for more information: 
```
if (isNaN(input)) {
    alert("Invalid input.");
  } else {
    return parseInt(input);
  }
```